benchmarks/popl18/nople/neg/BasicLambdas.hs

```diff
@@ -1,6 +1,6 @@
 
 {-@ LIQUID "--higherorder"      @-}
-{-@ LIQUID "--autoproofs"      @-}
+{- LIQUID "--autoproofs"      @-}
```

Comment:
Unrelated to this PR: I was never able to understand if such "unclosed" pragmas were:

* A genuine typo (i.e. somebody forgot the opening `@`);
* A leftover of the developer working on that test that needed a quick way to enable and disable the pragma,
  but eventually forgot about it.

I am wondering if we should somehow detect this (i.e. we have a block comment with a trailing `@-}` ) and emit
some sort of warning like "warning: undelimited LiquidHaskell pragma `--autoproofs`".

Reply:
Yes, it can be argued the flag should just be removed. I've kept it in the case someone who knows better than me what `autoproofs` is or was might still want to look at it in this context. Perhaps other occurrences of this pattern have similar history. It's certainly not good, and if @ranjitjhala or @nikivazou can say with confidence what the proper way for this negative test case to look like is now that `--autoproofs` does no longer exist, we can change it.

Reply:
Hi all, I think `autoproofs` should be replaced with `ple`